### ISSUE TITLE
Group Names for Announcement Messages

### ISSUE THREAD

**Issue by orionshock** [DeadlyBossMods/DBM-TBC-Classic#35]
Our guild has 7 Groups running Kara, and about to boot-up a 3rd 25man Team.
So things like this happen periodically:

> [07:36 PM] <DBM> Shade of Aran has been engaged by guild
> [07:37 PM] <DBM> Terestian Illhoof has been defeated by guild after 2 minutes and 20.64 seconds!

Would it be viable to some how add in a group name, as defined by the raid leader, or add in the Raid Leader's Name, something to indicate what group in the guild is doing what?

**Comment by MysticalOS** [comment]
Yes, eventually. but not for the forseeable future. retail is very busy
